proto/rpc/v1/account_info.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/ledger_objects.proto";
+
+// A request to get info about an account.
+message GetAccountInfoRequest {
```

Comment:
Are you talking about the RPC or the request type? I was following this style guide: [https://buf.build/docs/style-guide](https://buf.build/docs/style-guide), which says that the request types should be the RPC name suffixed with "Request".

If you are talking about the RPC, I prefer to prefix the RPC with a verb, similar to how a function would be named. On the client side, the RPC is called just like calling a function. However, I should probably change "Tx" to "GetTx" to be consistent. I am open to hearing other opinions about this.

Reply:
About the message name. I understand the Request/Response suffix, I'm partial to the Get prefix. If I had to choose, I'd remove it, but it might have some(?) value to keep it (and use it everywhere).

Reply:
I changed `Tx` to `GetTx`. I prefer the `Get` prefix.